dotnet-projects/StyleCop/StyleCop/GlobalSuppressions.cs

```diff
@@ -0,0 +1,9 @@
+﻿// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("StyleCop.CSharp.ReadabilityRules", "SA1137:Elements should have the same indentation",
+Justification = "<Pending>", Scope = "member", Target = "~F:StyleCop.Controllers.WeatherForecastController.Summaries")]
```

Comment:
Formatting as well : 
```suggestion
[assembly: SuppressMessage("StyleCop.CSharp.ReadabilityRules", "SA1137:Elements should have the same indentation",
    Justification = "<Pending>", Scope = "member",
    Target = "~F:StyleCop.Controllers.WeatherForecastController.Summaries")]

```